cms/static/js/models/validation_helpers.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Provide helper methods for modal validation.
+*/
+define(["jquery"],
+    function($) {
+
+        var validateIntegerRange = function(attributeVal, range) {
+            //Validating attribute should have an integer value and should be under the given range.
+            var isIntegerUnderRange = true;
+            var value = Math.round(attributeVal); // see if this ensures value saved is int
+            if (!isFinite(value) || /\D+/.test(attributeVal) || value < range.min || value > range.max) {
+                isIntegerUnderRange = false;
+            }
+            return isIntegerUnderRange;
+        }
+
+        return {
+            'validateIntegerRange': validateIntegerRange
+        }
```

Comment:
@mattdrayer  I introduced a new modal validation helper, here we can put common validation logic. if there is already any helper file then let me know.